automation/config/pod.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: best-app
+  labels:
+    app: best-app
+spec:
+  containers:
+  - name: best-app
+    image: manavrajvanshinx/best-app:latest
+    resources:
+      limits:
+        memory: 512Mi
+        cpu: "1"
+    env:
+    - name: DBHOST
+      value: db-test-svc
+    - name: DBPORT
+      value: '80'
+    - name: PASSWORD
+      valueFrom:
+        secretKeyRef:
+          name: db-secret
+          key: password
+    ports:
+    - containerPort: 3000
+  initContainers:
+  - name: init-db
+    image: busybox:1.28
+    command: ['sh', '-c', "until nslookup db-test-svc.$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace).svc.cluster.local; do echo waiting for database service; sleep 2; done"]
```

Comment:
this app/pod will only work for postgres type databases, this should be moved under pg tests directory, not re-usable by other containers.